### PR TITLE
SAAS-266 Platform SignOut

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/SAAS-266

- [x] https://github.com/percona-platform/saas/pull/12
- [x] https://github.com/percona/pmm/pull/568

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1049
API tests: https://github.com/Percona-Lab/pmm-api-tests/pull/120

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "revert-567-revert-566-SAAS-266-sign-out"
```
Comment (AlekSi) [inline Gopkg.toml:19]: Should be reverted

**Review by AlekSi — APPROVED** [review]

File: services/platform/platform.go [hunk 2]
```diff
@@ -144,6 +146,43 @@ func (s *Service) SignIn(ctx context.Context, email, password string) error {
 	return nil
 }
 
+// SignOut logouts that instance form Percona Platform account and removes session id.
```
Comment (palash25) [inline services/platform/platform.go:149]: `form` -> `from`

**Review by palash25 — APPROVED** [review]

File: services/platform/platform.go (hunk 2 above)
Comment (artemgavrilov) [reply services/platform/platform.go]: Arghhh... that typo follows me.

Fixed.

File: Gopkg.toml (hunk 1 above)
Comment (artemgavrilov) [reply Gopkg.toml]: Done

**Pull request merged**